endpoints_test.go

```diff
@@ -68,6 +76,156 @@ func TestHealthEndpoint(t *testing.T) {
 		t.Fatal("Error decoding JSON response from 'GET /health', ", err.Error())
 	}
 
-	assert.Equal(t, "UP 100ms", health.Database)
+	assert.Equal(t, "UP 100ms", health.Database)    //This assertion statement randomly fails from time to time
 	assert.Equal(t, viper.GetString("dummy_health_rabbitmq"), health.RabbitMQ)
 }
+
```

Comment:
Not 100% necessary, but it would be good to get some unhappy path tests in here too - we're currently only testing when things go well, but it's important that we're continuing to catch errors too. You don't have to go as over-the-top as I did in ras-rm-party and don't _necessarily_ have to add them now, though if not now, when? :D